### ISSUE TITLE
fail to build plotly python pkg due to use_2to3 is invalid

### ISSUE THREAD

**Issue by hubutui** [plotly/plotly.py#4193]
I failed to build plotly 5.14.1 with python 3.11, setuptools 67.7.0.
logs:
```text
/usr/lib/python3.11/site-packages/setuptools/config/setupcfg.py:293: _DeprecatedConfig: Deprecated config in `setup.cfg`
!!

        ********************************************************************************
        The license_file parameter is deprecated, use license_files instead.

        By 2023-Oct-30, you need to update your project and remove deprecated calls
        or your builds will no longer be supported.

        See https://setuptools.pypa.io/en/latest/https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

!!
  parsed = self.parsers.get(option_name, lambda x: x)(value)
/usr/lib/python3.11/site-packages/setuptools/dist.py:954: DistDeprecationWarning: use_2to3 is ignored.
  ep.load()(self, ep.name, value)
error in plotly setup command: use_2to3 is invalid.
```
quick fix, delete this line, https://github.com/plotly/plotly.py/blob/f006ae830365d6735e5d6fd3c3fb46dca10f539b/packages/python/plotly/setup.py#L498

**Comment by alexcjohnson** [comment]
closed by #4206 - thanks @hubutui and @J815!

**Comment by J815** [comment]
Welcome @alexcjohnson
